Api/V8/Config/routes.php

```diff
@@ -98,7 +98,7 @@
          */
         $app
             ->post(
-                '/module/{moduleName}/{id}/relationships',
+                '/module/{moduleName}/{id}/relationships/{linkFieldName}',
```

Comment:
Would it maybe make sense to leave the old route active as well?  
I think by changing the route it becomes a breaking change that's not backward compatible. Existing applications that use this API might stop working.  
It might be good to have both routes.

Reply:
That’s a good point, although personally I would argue it not to be a breaking change as the current route never worked as it should have. I don’t know if anyone would want an endpoint that creates a random link between two modules if they have more than one relationship. Well I believe it just uses the first/last link it finds. 

I do see the sense in maintaining backward compatibility and will be happy with whatever decision is made.